tex/chProgramming.tex

```diff
@@ -1138,9 +1148,10 @@ \subsection{Recursion on natural numbers}\label{ssec:recnat}
 % second argument is 0; but this
 rule thus also covers the case where the
 second argument is non-zero while the first argument is 0: in that
-case, the result of the function is zero.  Another possible view on
-\C{subn} it to see it as a subtraction operation on natural numbers,
-made total by providing a default value in the ``exceptional'' cases.
+case, the result of the function is zero.  We can view
```

Comment:
I don't approve this change

Reply:
The change goes on for a few more lines -- I reworded rather than removed this sentence. I was trying to remove the word "exceptional" since I found it somewhat counterintuitive to think of 1/2 of all possible inputs as "exceptional". But I don't have a strong preference either way.

Reply:
Another way to state this behavior precisely is saying that the subtraction is truncated at 0. See [monus](https://en.wikipedia.org/wiki/Monus).